### DISCUSSION TITLE
Show rust backtrace

### DISCUSSION THREAD

**Discussion by rherrmannr** [tauri-apps/cargo-mobile2#126] [Q&A]
I got a crash in my android emulator and want to see the backtrace. I only get this message:
```
04-02 21:58:55.147  4716  4770 I RustStdoutStderr: note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
I'm running it locally with `cargo tauri android dev`. I've tried with `-v` and `-vv` for verbose output but I want to have the backtrace from the executed rust code.

**Comment by c608345** [comment]
RUST_BACKTRACE=1  cargo tauri android dev
